src/contracts/test/TransferExecutor.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../libraries/GPv2SafeERC20.sol";
+
+contract TransferExecutor {
```

Comment:
naming uber-nit: Maybe `GPv2SafeERC20TestInterface` to match naming scheme of other test contracts?